src/key.h

```diff
@@ -183,7 +183,10 @@ struct CExtKey {
     template <typename Stream>
     void Unserialize(Stream& s)
     {
-        unsigned int len = ::ReadCompactSize(s);
+        unsigned int len = ::ReadCompactSize(s); //not using BIP32_EXTKEY_SIZE as max size here -> want to throw my own exception
+        if (len != BIP32_EXTKEY_SIZE) {
```

Comment:
BIP32_EXTKEY_SIZE we can also make size_t and convert to constexpr:
constexpr size_t BIP32_EXTKEY_SIZE = 74;